src/main/java/com/enfint/dossier/email/CustomEmailMessages.java

```diff
@@ -0,0 +1,123 @@
+package com.enfint.dossier.email;
+
+import com.enfint.dossier.dataEnum.Theme;
+import com.enfint.dossier.dto.ApplicationDTO;
+import com.enfint.dossier.dto.EmailMessage;
+import com.enfint.dossier.fiegnClient.DealClient;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+
+@Service
+@RequiredArgsConstructor
+public class CustomEmailMessages {
+
+    private final DealClient dealClient;
+    public String sendCustomMessage(EmailMessage emailMessage, String subject ){
+        Theme theme = emailMessage.getTheme();
+        Long applicationId = emailMessage.getApplicationId();
+        ApplicationDTO application = dealClient.getApplicationById(applicationId);
+        String firstName = application.getClient().getFirstName();
+        String link = "http://localhost:8085/document/";
+        if (emailMessage.getTheme() == Theme.SEND_DOCUMENTS)
+            dealClient.updateApplicationStatus(emailMessage.getApplicationId());
+
+        return "<div style=\"font-family:Helvetica,Arial,sans-serif;font-size:16px;margin:0;color:#0b0c0c\">\n" +
```

Comment:
It is better to move this HTML in separate file. 
Also you can use "Text blocks" feature in since Java 15 
`String html = """
              <html>
                  <body>
                      <p>Hello, world</p>
                  </body>
              </html>
              """;`

Reply:
https://openjdk.org/jeps/378